modules/passkey/test/utils/webauthn.ts

```diff
@@ -357,3 +357,34 @@ export function encodeWebAuthnSignature(response: AuthenticatorAssertionResponse
     [new Uint8Array(response.authenticatorData), clientDataFields, r, s],
```

Comment:
nit: This should be re-written with `getSignatureBytes`.

Reply:
done

Reply:
This change actually led to failing tests with `TestWebAuthnSigner` :see_no_evil: